src/base/static/components/organisms/main-map.js

```diff
@@ -87,12 +87,6 @@ class MainMap extends Component {
     // Instantiate the map.
     this._map = MapProvider(props.container, props.mapConfig.options);
 
-    // Set initial map position state.
-    this.props.setMapPosition({
-      center: this.props.mapConfig.options.map.center,
-      zoom: this.props.mapConfig.options.map.zoom,
-    });
-
```

Comment:
Note that this had to be moved to AppView because the MapPosition is being accessed in AppView, so it should be treated as a global state.

See the calls to `mapPositionSelector` in the `viewPlace` method of AppView where this state is needed.

There are some other pain points in MainMap that we should consider cleaning up, like the event listeners. Now that we are using Redux, I think we can refactor those listeners into Redux actions, which should save us the extra work of coordinating the registering/de-registering of the listeners.